facades/PC/build.gradle.kts

```diff
@@ -69,22 +67,27 @@ val displayVersion = versionBase
 
 
 application {
+    applicationName = "Terasology"
     mainClass.set(extra.get("mainClassName") as String)
 }
 
-// Adjust as the Gradle 6 upgrade changed this path a bit
-sourceSets {
```

Comment:
I guess this doesn't get in the way of anything? Like runtime hot reloading or stuff like that. Not really sure what this would affect. Assets changing at runtime is one thing but unsure if that'd be affected.

Reply:
facade's sources don't contain any gestalt modules, so we don't have to be as controlling when we build this as we do when we build modules.